getting-started/mag2exp.ipynb

```diff
@@ -0,0 +1,470 @@
yYoVK/jhhx9ITY0oUz169GDy5Mmy3xYVFTFy5Ei6d+9OY2Mjdru91T9dVFRE9+7diY+PJy0tjf3797earIuLi+nQoQOxsbF0796d/fsjZEibzUZTUxM9evSgqalJ856/B05YH7O95dgmuv/oxXpFm85iJljfhCE1Yo48VOtvbimlolpbSB4uGmp20dKkzrZtmrcZxxZ1ge89hsn+P3xwL47GAEZzZEjFpZgI2prxFBQQ1bu3/Jlt+Xh92iFTRwuDOYqW0l2KdikkIqnsFqH9XeaR4PuXSqjId2G2RoRddLwB3/4S3Lvz0EdHYeoYqSDW0lJKMPjrCDihUED2/IJeT9KIsVR89xH6KLki0/GUJKr3Hps+b6oPMu0rdS6GqVMmoUYlOc7uqSK/cpGi/WjhdFRRX7sTkzkWs/kQZUWUqP5iCYbEGCzZcreGJEk4Hcd27v+vobrUz/fvKderPxKrVq1i5syZfP755zKhDDBu3DhKS0tZsyZMDFyxYgXV1dWcc8455Obm0r9/fz7//HMgXA959uzZXHnllQBcdtllfPHFF/j9fvx+P1988YXs2MyZM6mpCRNWP/vsM0499VSys7PbvefvgRNWMH/1gR2n/cgmqNYCPvv7FhbPcLBqgZKME6xtxJCuNJ15vE3UNe4iNVmZZEOSJMr8ewi1YR6rhccARMWksXPTZ7TYShTH9FFmKt6YTqBBuSD/8IUTm0ou6PZQWBygtFxJEjr13BRm/LeUkp1ys1LyZZfSNHs2Yps6vUaDle37vqW6fhuBoPy97IEGHEHtSktB0U+hbS3eNuUHXXVllK/6GaNVacXY8f5GvE3q/qF5c49MOFZUBtm8TUmYG35hClNeL2Pn8ohQEgSBpOuupmX2fIIt8m/g8znYtv0zWlrKcDjkrPmg6KfRU6bKPgcIBL0UV62guHq5TNkLOuyUf/0+OqMJnUG+o9u3tJrd89QTXNTUHtk42LjCzTfvNbF0lnLMGzPT8VcpQ9OspkRa3JU0u5TkOEkSNd9Vi53fbCumtmab8vfVTQRtTpqXKxW06ooQM35U34n/fwqjkiSJL791sW2Hcpzu2eJm6fQWtq5WD00TJOlX/x0pPv/8cxwOB9deey3nn39+69/evXtJSkrinXfe4YUXXmDs2LG8+uqrvPfeeyQkJADw+uuvs23bNs444wxuuukm7r//fkaMCJcZvf/+++nUqRNnn30248aNo1u3btx7770AnH766UyYMIHrr7+eM844gz179vDaa68B/OI9f2uccKbsg9i2yUfKzy6uvV25iM+c7+Li85T+0KlzXJzS10zXXLlJzBol0FATJLWDvDtDThf+0ioMqUmtdXEPosFWQGXNRmKi0kmMz5WdVxUsZH9gF4n6DsTqI7ttt2hni3MBmaauZJp7YBTCZraomHQkSaQobzoDhk3AYIwQRYxp8QhGA0GHG2OK3CdoaxJ58UkbL72rNCvv3uejb08la3P+Yg9Wi8AdN7dhoNsCrPyxlszuUeT2jxwzxMcTO2QITdNnED1oINau4WxV8bGd6NhhGLsLp5CdMZIeuRdE3t9XQKlnBwmGdHKiBpBu6owgCIiSSIVjJ4W2NaRHdcPSJvOX11aD3mTFECV/tuaiJuz7bcTlJijeJ39fgE8/dnHBhUrf+Iz5Lk4baiE5SW7u3bI9wNwFHga/KSeh+X0iu1c3Y7Lq6D8m8t300VEkXnEJjd98T/SppxAzYhgAaWn9iI5OY8u2TzAarAwd8mf0+vA3LWheS5l9GzrBQFpUZ7oljCLaGH5+p6eOTfu+QJJCnNbvfvlD6wQEnZ7oLj1kzT5ngLn/2YUYVCqjoijxyBPNTPpIqUDuLwsgSdAlRz7mE1P0NNSEiG5TyEOSJAwZafiLyzF3zkYfF/lG9S37sLuraHaVkxAtJ6fZqMcv+eggKNOZFgW309MwBF0bc2+zrZiW5hK83mYsloTW9kCjA1NWElJQKeh3bvUz/QcX198eo/BZf/2jkxuujFGwpb1ekaLSgOp8mLvYzZiRFoW/eX9ZgPyiAOedqeQUzF/qZuQQC3Ft+k4UJXbt9TOgj/I+S1e7ye1kpHO2UXHOnEVuLjpXvl4JgsCevQES4nUMHCA7hN8nUl7ow2zR2Jf9AfrJp59+2u7xkSNHMmvWLNVjWVlZrTvmtjCZTDz77LOa173lllu45ZZbjvievzVOyB2zKEoU7vHz01cO1V1wcqK63+38M6MUQhmg/xArOd1N9Gibgzco0jJtMc6lG6DNIiBJImZTHOmpA9q0S1iEaE6PulImlAESDemkGjtR49+P4RCdSqfT02vANYCAro3/Lu2a04kb0ZOQQ7nbFgRYNNvDojnKnWSXbHV/3H13xTJqhFKInXFtB/qOTqCuVBk3qY+Lx7F2Le6dO2XtKQk9yM4YRWXdFlyeSAhNsjGLfrFj6RN7Bmmm3NYFVABqXfmIYpCuiSMU9/E7bWSNupT4HDmRKqFrElEZsQx/8kzFOZ9+7CY/P6g6Di46J0ohlAEuudDKPXcp/d09hsRxyZ87UZan3JHpExPwV1bRPHeBLBGM0RhNx6wRiJJIfkFkEciNG0RmTC/6JY+jT9JZrUJZkiRqbXuwmhLomnkmJqN8QXYX5xPTawCp4y6RtRvMenKHJDP8JmUaz115AXbtVo/d75xtJLeTUn8fekYUvQea6dBRJXRu9Sbsc5YgtslQlpk8kGhziiKVKkCSkE6yoEzFJkoibsmBR1Lu7jKyhpGa3v8QfkUYMQNyie7diZiBytS3O7f6KNwbYM9O5fvGxepwupSKi9ks0LOruq/57NOtqiSwzHQDZ56mzqQ+bZhSKAPodAL9eqnfZ+woq0IoHzzngrPVs5b9/cFYuqqkWj3zsgTikvRkd1cPlzpZXeqPxwm5Y/Z6JG66O464eB1+H7TNdjlqmHoqQC0WZmqGkVv+qgyx0sXHIERZiB03QnFMkkR6d78co0E+eQVBINmQiRa6WAbSydxb
```

Comment:
Is this important information for x-ray holography in this introductory notebook? (For LTEM I assume it is because it would determine the out-of-focus direction?)